codec-mqtt/src/test/java/io/netty/handler/codec/mqtt/MqttCodecTest.java

```diff
@@ -500,36 +801,57 @@ private static MqttMessage createMessageWithFixedHeaderAndMessageIdVariableHeade
     }
 
     private static MqttConnectMessage createConnectMessage(MqttVersion mqttVersion) {
-        return createConnectMessage(mqttVersion, USER_NAME, PASSWORD);
+        return createConnectMessage(mqttVersion,
+                USER_NAME,
+                PASSWORD,
+                MqttProperties.NO_PROPERTIES,
+                MqttProperties.NO_PROPERTIES);
     }
 
-    private static MqttConnectMessage createConnectMessage(MqttVersion mqttVersion, String username, String password) {
+    private static MqttConnectMessage createConnectMessage(MqttVersion mqttVersion,
+                                                           String username,
+                                                           String password,
+                                                           MqttProperties properties,
+                                                           MqttProperties willProperties) {
         return MqttMessageBuilders.connect()
                 .clientId(CLIENT_ID)
                 .protocolVersion(mqttVersion)
                 .username(username)
-                .password(password)
+                .password(password.getBytes())
```

Comment:
Is it safe to rely on "default" encoding for these tests?

Reply:
Changed to `password.getBytes(CharsetUtil.UTF_8)`